### PR TITLE
reference npm over yarn across `if` repo

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request
We were inconsistent in recommending yarn and npm across various parts of the repository., This PR stanbdardizes the repository to use npm.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -4,35 +4,28 @@
...
+			"extensions": [
+				"esbenp.prettier-vscode",
+				"ms-vscode.makefile-tools"
+			]
+		}
+	},
 	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
 	// "remoteUser": "root"
-}
+}
```
Comment (narekhovhannisyan) [inline .devcontainer/devcontainer.json:31]: Missing newline at the end of file. I think this changes can be reverted.

File: .github/workflows/nodejs-ci.yml [hunk 2]
```diff
@@ -14,13 +14,13 @@ jobs:
...
-        run: yarn install
+        run: npm install
 
       - name: Run lint
-        run: yarn lint
+        run: npm run lint
 
       - name: Run tests
-        run: yarn test --silent
+        run: npm run test
```
Comment (narekhovhannisyan) [inline .github/workflows/nodejs-ci.yml:26]: npm run test -- --silent

File: package.json [hunk 3]
```diff
@@ -62,9 +61,9 @@
...
     "impact-engine": "npx ts-node src/index.ts",
     "lint": "gts lint",
-    "prepublish": "yarn build",
+    "prepublish": "npm run build",
     "test": "jest --verbose"
   },
   "stability": "stable",
   "types": "src/index.d.ts"
-}
+}
```
Comment (narekhovhannisyan) [inline package.json:69]: missing newline at eof

File: .github/workflows/nodejs-ci.yml (hunk 2 above)
Comment (jmcook1186) [reply .github/workflows/nodejs-ci.yml:26]:
```suggestion
        run: npm run test --silent
```

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (jmcook1186) [reply .devcontainer/devcontainer.json:31]:
```suggestion
}

```

**Review by narekhovhannisyan — APPROVED** [review]

**Comment by narekhovhannisyan** [comment]
lgtm

**Pull request merged**